addons/account/models/account_payment.py

```diff
@@ -184,10 +184,10 @@ def _seek_for_lines(self):
         writeoff_lines = self.env['account.move.line']
 
         for line in self.move_id.line_ids:
-            if line.account_id in self._get_valid_liquidity_accounts():
-                liquidity_lines += line
-            elif line.account_id.internal_type in ('receivable', 'payable') or line.account_id == line.company_id.transfer_account_id:
+            if line.account_id.internal_type in ('receivable', 'payable') or line.account_id == self.company_id.transfer_account_id:
                 counterpart_lines += line
+            elif line.account_id in self._get_valid_liquidity_accounts() or line.account_id.reconcile:
```

Comment:
I don't know. The question is: is it possible to have a line that should normally be classified as writeoff and which account's allows reconciliation ? I guess it's again a question for @smetl

Reply:
I don't know but I tend to say yes. Anyway, to me the flow is weird, I would like to have the opinion of the POs about this one and another ticket involving the same lines of the code. Sorry for the waste of time, should have put this one in draft

Reply:
Oh, ok, let us know what the PO think about this one :)

Reply:
@guva-odoo It's not the first time we try to solve something like that. Maybe we should do something more flexible when the payment has only two accounts.
- If one is receivable/payable/transfer, the other is the liquidity one
- If one is the liquidity one, the other is the counterpart one
- else: what we already have now